beacon_node/network/src/sync/block_lookups/tests.rs

```diff
@@ -1287,19 +1317,6 @@ fn test_skip_creating_failed_parent_lookup() {
     rig.expect_no_active_lookups();
 }
 
-#[test]
-fn test_skip_creating_failed_current_lookup() {
```

Comment:
Not applicable anymore. A peer makes no claim that it has imported the child block, so it should not be downscored